pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBuilderImpl.java

```diff
@@ -151,7 +151,7 @@ public CompletableFuture<Consumer<T>> subscribeAsync() {
                     Math.max(conf.getBatchReceivePolicy().getMaxNumMessages(), conf.getReceiverQueueSize()));
         }
         CompletableFuture<Void> applyDLQConfig;
-        if (conf.isRetryEnable() && conf.getTopicNames().size() > 0) {
+        if (conf.isRetryEnable() && !conf.getTopicNames().isEmpty()) {
```

Comment:
This change is not related to the PR. It's better to revert it or use a separated PR for such small trivial changes.

Reply:
Updated.